src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -61,16 +63,33 @@
 @Builder(toBuilder = true)
 public class CaseData {
 
-    @CCD(access = {DefaultAccess.class, CaseworkerWithCAAAccess.class})
+
+    @CCD(
+        label = "Launch the Flags screen",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private FlagLauncher flagLauncher;
```

Comment:
since this has been moved from cicCase (which is has all fields prefixed with cicCase) you will need a migration to handle the new name

Reply:
Added the fields that are removed from CicCase to RetiredFields.